oggm/core/gcm_climate.py

```diff
@@ -16,15 +20,12 @@
 
 
 @entity_task(log, writes=['gcm_data', 'climate_info'])
-def process_gcm_data(gdir, filesuffix='', prcp=None, temp=None,
-                     time_unit='days since 1801-01-01 00:00:00',
+def process_climate_data(gdir, filesuffix='', prcp=None, temp=None,
+                     time_unit='days since 1800-01-01 00:00:00',
```

Comment:
I think we should remove this time_unit keyword altogether - I'll have a look at what I can do

Reply:
Or it should be determined from the climate dataset we are using

Reply:
@fmaussion, @nchampollion could you let me know if you remove the `time_unit` keyword before merging the PR? So I can have a look :)